query/language/define.feature

```diff
@@ -980,13 +1089,32 @@ Feature: TypeQL Define Query
       | values("1", "2") |
 
 
+    # Uncomment and fill when annotations with arguments appear for entity types
```

Comment:
Don't put TODOs here because it's an optional step, but I want to preserve the structure of the file. Plus, it may remind us about these steps missing even if we ignore it at first.